.husky/pre-push

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Does this need single quotes?

Reply:
Since this is not a .ts file (if i'm correct this is a bash script) this should be in double ""